Model/Indexer/Data.php

```diff
@@ -94,6 +142,23 @@ public function executeFull()
         }
     }
 
+    /**
+     * @param DimensionProviderInterface $dimensionProvider
+     * @param array $ids
+     * @suppress PhanTypeMismatchArgument
+     */
+    private function executeInParallel(DimensionProviderInterface $dimensionProvider, array $ids = [])
+    {
+        $userFunctions = [];
```

Comment:
Remove this method. All the logic for parallelization should be in the `executeByDimensions` method.